backend/package.json

```diff
@@ -23,6 +23,7 @@
     "drizzle-orm": "^0.30.4",
     "express": "^4.18.3",
     "iron-session": "^8.0.1",
+    "knex": "^3.1.0",
```

Comment:
Doesn't look like you're using this package so you should remove it with 
`yarn remove knex`